src/css/main.css

```diff
@@ -587,8 +587,8 @@ span.subcomponent {
   cursor: grabbing;
 }
 
-code, pre {
-  font-family: Consolas, Andale Mono, Monaco, Courier New, monospace;
+.wf-active code, .wf-active pre, .wf-active input, .wf-active textarea, .wf-active select {
```

Comment:
this is correct, but I would suggest putting each selector on a newline to improve readability